packages/fxa-settings/src/pages/Signin/SigninRecoveryMethod/index.tsx

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import AppLayout from '../../../components/AppLayout';
+import { HeadingPrimary } from '../../../components/HeadingPrimary';
+import FormChoice, {
+  CHOICES,
+  FormChoiceData,
+  FormChoiceOption,
+} from '../../../components/FormChoice';
+import { useFtlMsgResolver } from '../../../models';
+import {
+  BackupAuthenticationCodesImage,
+  BackupRecoveryPhoneSmsImage,
+} from '../../../components/images';
+import ButtonBack from '../../../components/ButtonBack';
+
+const SigninRecoveryMethod = () => {
+  const onSubmit = async ({ choice }: FormChoiceData) => {
+    // TODO: actually do something with this, maybe pull into container
+    console.log('Submitted with choice:', choice);
+  };
+
+  const ftlMsgResolver = useFtlMsgResolver();
+  // TODO, actually pull these values
+  const numberOfCodes = 4;
+  const lastFourPhoneNumber = 3019;
+
+  const formChoices: FormChoiceOption[] = [
+    {
+      id: 'recovery-choice-phone',
+      value: CHOICES.phone,
+      image: <BackupRecoveryPhoneSmsImage />,
+      localizedChoiceTitle: ftlMsgResolver.getMsg(
+        'signin-recovery-method-phone',
+        'Recovery phone'
+      ),
+      // This doesn't need localization
+      localizedChoiceInfo: `••••••${lastFourPhoneNumber}`,
+    },
+    {
+      id: 'recovery-choice-code',
+      value: CHOICES.code,
+      image: <BackupAuthenticationCodesImage />,
+      localizedChoiceTitle: ftlMsgResolver.getMsg(
+        'signin-recovery-method-code',
+        'Authentication codes'
+      ),
+      localizedChoiceInfo: ftlMsgResolver.getMsg(
+        'signin-recovery-method-code-info',
+        `${numberOfCodes} of 5 available`,
```

Comment:
Noticed that the number of generated codes is controlled by an env variable. The default is 8, but the actual value varies per environment (and could theoretically change). Perhaps we should instead have "X codes remaining" without reference to the initially generated number?

https://github.com/mozilla/fxa/blob/4e7c4140f90f7bbda51f473fe1f07281131f10a7/packages/fxa-content-server/server/lib/configuration.js#L705-L711

Reply:
Thank you for catching this! Yes, looks like [Jeff confirmed](https://mozilla.slack.com/archives/C07T1GHL46S/p1736200471677559?thread_ts=1736199262.345029&cid=C07T1GHL46S) we should change this to "X codes remaining".

Reply:
Consulted with UX, decision is to go with "X codes remaining". https://mozilla.slack.com/archives/C07T1GHL46S/p1736200471677559?thread_ts=1736199262.345029&cid=C07T1GHL46S